README.md

```diff
@@ -76,3 +76,360 @@ be found in [`lib/transformations.js`](lib/transformations.js).
 ## Distributed Metamodel Repository
 
 ## Data Access Layer
+
+## Interface: fs.provider
+
+### 
+
+`FsProvider()`
+
+
+### 
+
+`FsProvider.prototype.constructor()`
+
+
+### 
+
+`FsProvider.prototype.readStat()`
+
+
+### 
+
+`FsProvider.prototype.open()`
+
+
+### 
+
+`FsProvider.prototype.writeStat()`
+
+
+### 
+
+`FsProvider.prototype.close()`
+
+
+### 
+
+`FsProvider.prototype.takeId()`
+
+
+### 
+
+`FsProvider.prototype.get()`
+
+
+### 
+
+`FsProvider.prototype.create()`
+
+
+### 
+
+`FsProvider.prototype.update()`
+
+
+### 
+
+`FsProvider.prototype.delete()`
+
+
+### 
+
+`FsProvider.prototype.select()`
+
+
+## Interface: pg.provider
+
+### 
+
+`PostgresProvider()`
+
+
+### Create PostgresProvider
+
+`PostgresProvider.prototype.constructor()`
+
+
+### Open PostgresProvider
```

Comment:
I thought this would be the other way around:

Function/method name as a title
description below

i.e.

-----

### `PostgresProvider.prototype.open(options, callback)`
   // some version descriptions
   - `options: `[`<Object>`] to be passed to pg
   - `callback: `[`<Function>`]
     - `err: `[`<Error>`]` | `[`<null>`]
     - `provider: `[`<this>`] 

Open PostgresProvider

-----
This will give us a meaningful links later on and allow to easily auto-generate the navigation.
I don't think we need description-as-a-title at all, it will only confuse the reader IMO.

Reply:
I don't think function/method name should be a level 3 header.